src/IssueViz/IssueVisualizationControl/ViewModels/IssueVisualizationViewModel.cs

```diff
@@ -68,6 +69,25 @@ internal sealed class IssueVisualizationViewModel : INotifyPropertyChanged, IDis
                 selectionService.SelectedLocation);
         }
 
+        public string IssueLocation
+        {
+            get
+            {
+                var issueSpan = CurrentIssue?.Span;
+
+                if (issueSpan == null || issueSpan.Value.IsEmpty)
+                {
+                    return null;
+                }
+
+                var position = issueSpan.Value.Start;
+                var line = position.GetContainingLine();
+                var fileName = Path.GetFileName(CurrentIssue.CurrentFilePath);
+
+                return $"{fileName}:{line.LineNumber}";
```

Comment:
I have a very bizarre behavior: the line number is (-1).
This is the exact same code that we use in IssuesSnapshot, yet there the line number is correct.
I debugged and it seems like when the span changes:
1. the error list gets notified like 20 times
2. the panel gets notified
3. the error list gets notified _again_, this time with the updated line.

I'm not sure what is happening.

Reply:
What do you mean the line number is (-1)? It's one lower than you expect?

I think the line number is useful information, and the label is a bit confusing: `Source1.cpp, line 3` would be clearer.

Reply:
One lower, yes. So the error list would say "line 5" - and panel "line 4"

Reply:
It looks a bit weird with primary location above the description; it's a location, but it's separated from the other locations, and doesn't look the same (no file type icon).